### DISCUSSION TITLE
https services questions

### DISCUSSION THREAD

**Discussion by marianne013** [DIRACGrid/DIRAC#6994] [Q&A] [answered]
Hi,

@sfayer and I just had a look at enabling https services on one of our pre-prod servers (v8.0.18).
So far we enabled it for JobManager, JobStateUpdate and JobMonitor and it seems to work.
Having said this @sfayer wonders how this compares to the previous single services wrt request/s performance ? We have a very busy file catalog on our production server and we worried those requests might swamp out other services in tornado.

Also is there a way to restart/disable a single service like in the old system ? And if so how ?

Will there be any attempts at splitting the logs on disk up again, or is it "elastic/open/whatevercomesnext/search or bust" ?

Cheers,
Daniela

**Comment by chaen** [comment, marked as answer]
Hi,

>how this compares to the previous single services wrt request/s performance ? 

Last time this was checked (before oauth refactoring), HTTPs was better

> Also is there a way to restart/disable a single service like in the old system ? And if so how ?
Not so sure what you mean, but the baseline is that all tornado processes run all services

> Will there be any attempts at splitting the logs on disk up again, or is it "elastic/open/whatevercomesnext/search or bust" ?

No, nothing is planned, but it does not mean you cannot do it. It's just a different log handler.

That being said, I would strongly advice not to run any HTTPs service until this is fixed https://github.com/DIRACGrid/DIRAC/issues/6877

**Reply by marianne013** [reply]
> > Also is there a way to restart/disable a single service like in the old system ? And if so how ?
> > Not so sure what you mean, but the baseline is that all tornado processes run all services

I was trying to remove Janusz' pilot logging that had still been on the node from a previous testing attempt, as we only wanted to test one thing at a time. In the end I deleted the config and then restarted tornado, which did the trick, but I was hoping there was a way to disable it, so I wouldn't have to put the config back when we get back to this particular can of worms.

**Reply by chaen** [reply]
All the services which have `Protocol=https` will be served, so you can just edit this part of the config
